client/src/pages/signup.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Button, TextField, Paper, FormControl } from '@material-ui/core';
+import { Link } from "react-router-dom";
+import laptopPhoto from '../images/blue-shirt-at-laptop.png'
+import './signup.css'
+
+export default function Signup() {
+
+    const [firstName, setFirstName] = useState("");
+    const [lastName, setLastName] = useState("");
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [confirmPassword, setConfirmPassword] = useState("")
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        if (password === confirmPassword) {
+            axios.post('/signup', {
+                firstName: firstName,
+                lastName: lastName,
+                email: email,
+                password: password
+            })
+                .then(window.location = "/")
+        } else {
+            alert("Passwords do not match")
```

Comment:
this can be discussed with Jess and your team about when you want to implement a better solution, but `alert` should definitely not be the long term solution here.